converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/ChangeFeedProcessor.cs

```diff
@@ -114,7 +114,7 @@ public async Task ProcessAsync(TimeSpan pollIntervalDuringCatchup, CancellationT
                                 errorType,
                                 cancellationToken);
 
-                            _logger.LogError("Failed to process DICOM event with SequenceID: {SequenceId}, StudyUid: {StudyUid}, SeriesUid: {SeriesUid}, instanceUid: {InstanceUid}  and will not be retried further. Continuing to next event.", changeFeedEntry.Sequence, studyUid, seriesUid, instanceUid);
+                            _logger.LogError("Failed to process DICOM event with SequenceID: {SequenceId}, StudyUid: {StudyInstanceUid}, SeriesUid: {SeriesInstanceUid}, instanceUid: {SopInstanceUid}  and will not be retried further. Continuing to next event.", changeFeedEntry.Sequence, studyInstanceUid, seriesInstanceUid, sopInstanceUid);
```

Comment:
I would just confirm once if the property name is being matched with IgnoreCase. If not, just double check if the property name should be StudyInstanceUid or studyInstanceUid.

Reply:
Same goes for other property names as well.

Reply:
As we talked offline, looks like it is not ignore case, but it needs to be uppercase here so will handle on redaction side.